src/tools/perf/cuda/ucp_cuda_kernel.cu

```diff
@@ -183,16 +183,19 @@ private:
         params.elements     = elems;
 
         ucs_status_t status;
-        const ucs_time_t deadline = ucs_get_time() + ucs_time_from_sec(5.0);
+        ucs_time_t deadline = ucs_get_time() + ucs_time_from_sec(60.0);
         do {
+            if (ucs_get_time() > deadline) {
+                ucs_device_debug("Timeout on creating device memory list has been exceeded.");
```

Comment:
1 . it should be ucs_warn since it's host code, and we want it to be visible

```suggestion
                ucs_warn("timeout creating device memory list");
```